CloudVision_Studios/RCF Studio/Readme.md

```diff
@@ -0,0 +1,2 @@
+RCF (Routing Control Function) Studio is a tool designed for the management of code units across multiple devices.
```

Comment:
Routing Control Function (RCF) Studio allows defining multiple code units and use them across devices in the network.

The general RCF language structure is documented here: https://www.arista.com/en/support/toi/eos-4-27-2f/15102-routing-control-functions-language-and-configuration
All attributes and builtin functions are documented here: https://www.arista.com/en/support/toi/eos-4-27-2f/15101-routing-control-functions-attribute-and-builtin-function-reference